sectionsUtilityTool/src/main/webapp/assets/js/controllers/controllers.js

```diff
@@ -14,7 +14,7 @@ sectionsApp.controller('termsController', ['Courses', '$rootScope', '$scope', '$
   $scope.selectedTerm = null;
   //reset term scope
   $scope.terms = [];
-  var termsUrl ='manager/api/v1/accounts/1/terms?per_page=4000';
+  var termsUrl ='manager/api/v1/accounts/1/terms?per_page=4000&_=' + (new Date().getTime());
```

Comment:
Maybe make the date / time code  into a function that creates the query parameter and give it a clear name.  E.g. avoidCachingViaParameter?